C24_WMDE_Desktop_DE_01

Sperling-0 opened the pull request:
- The banners are based on the control banner of desktop-de-25 (2023)
- Following changes are for both the banners:
         - The progress bar is removed
         - The soft-close feature is removed/not used
         - Closing the banner should set a cookie to prevent banner display for 7 days
         - Closing the banner should trigger a banner closed event
         - The already-donated feature is removed/not used
         - The visitors-vs-donors sentence is removed
         - The campaign sentence is not removed
         - The copy is changed
         - The feature to redirect users directly to the payment providers is removed
         - The removed elements are made up for, so the banners share the same height with the base (campaign) banner
         - The larger fallback banner uses a different highlighted sentence
         - The progress bar is removed from the bigger+sliding fallback banner

Ticket: https://phabricator.wikimedia.org/T366159

Sperling-0 committed:
Prepare campaign for C24_WMDE_Desktop_DE_01

- The banners are based on the control banner of desktop-de-25 (2023)

Ticket: https://phabricator.wikimedia.org/T366159

Sperling-0 committed:
Change CTRL banner for precampaign

- The progress bar is removed from CTRL banner
- The soft-close feature is removed/not used
- Closing the banner should set a cookie to prevent banner display for 7 days
- Closing the banner should trigger a banner closed event
- The already-donated feature is removed/not used
- The visitors-vs-donors sentence is removed
- The campaign sentence is not removed
- The copy is changed
- The feature to redirect users directly to the payment providers is removed
- The removed elements are made up for, so the banners share the same height with the base (campaign) banner
- The larger fallback banner uses a different highlighted sentence
- The progress bar is removed from the bigger+sliding fallback banner

Ticket: https://phabricator.wikimedia.org/T366159

Abban commented on banners/desktop/C24_WMDE_Desktop_DE_01/banner_ctrl.ts:
```diff
@@ -0,0 +1,76 @@
+
+app.provide( 'currencyFormatter', currencyFormatter );
+app.provide( 'formItems', createFormItems( translator, currencyFormatter.euroAmount.bind( currencyFormatter ) ) );
+app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount ) );
```
```suggestion
app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount, { ap: '0' } ) );
```

Abban commented on banners/desktop/C24_WMDE_Desktop_DE_01/banner_var.ts:
```diff
@@ -0,0 +1,76 @@
+
+app.provide( 'currencyFormatter', currencyFormatter );
+app.provide( 'formItems', createFormItems( translator, currencyFormatter.euroAmount.bind( currencyFormatter ) ) );
+app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount ) );
```
```suggestion
app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount, { ap: '1' } ) );
```

Abban reviewed (commented):
Here's what you need to change to show the correct forms on the application

Sperling-0 committed:
Implement VAR for C24_WMDE_Mobile_DE_01

- The variant banner redirects to the variant donation form
- It uses BannerCtrl.vue

Ticket: https://phabricator.wikimedia.org/T366159

moiikana commented on campaign_info.toml:
```diff
@@ -6,24 +6,24 @@
-tracking = "org-00-190624-var"
+filename = "./banners/desktop/C24_WMDE_Desktop_DE_01/banner_var.ts"
+pagename = "B24_WMDE_Desktop_DE_01_var"
+tracking = "org-01-010724-var"
```
got the info from Kai today that the date format should be yymmdd, I had to change it in the mobile banner too
-> 240701

moiikana commented on banners/desktop/C24_WMDE_Desktop_DE_01/content/BannerSlides.vue:
```diff
@@ -1,40 +1,37 @@
 		<p>
-			Die meisten Menschen spenden, weil sie Wikipedia nützlich finden. Die durchschnittliche Spende
-			beträgt 22,25&nbsp;€, doch bereits 5&nbsp;€ helfen uns weiter.
+			Die durchschnittliche Spende beträgt xx,xx €, doch bereits 5 € helfen uns weiter. Hat Wikipedia Ihnen in
```
Kai updated the numbers in the ticket on Wednesday, need to be updated
